app/views/DR/Maps/Map.js

```diff
@@ -24,6 +24,7 @@ import {
   requestCovid19Hospitals,
   requestCovid19Laboratories,
 } from '../../../helpers/Request';
+import languages from '../../../locales/languages';
```

Comment:
Please use the hooks **useTranslation** instead of languages because in that form we use the same instance of the language and the label will change at the same time with the others
```suggestion
import { useTranslation } from 'react-i18next';
```